src/IntelliTect.Coalesce/TypeDefinition/ParameterViewModel.cs

```diff
@@ -39,7 +40,7 @@ public ParameterViewModel(MethodViewModel parent)
         public string DisplayName =>
             this.GetAttributeValue<DisplayNameAttribute>(a => a.DisplayName) ??
             this.GetAttributeValue<DisplayAttribute>(a => a.Name) ??
-            Name.ToProperCase();
+            Name.ToProperCase()!;
```

Comment:
Is null forgiveness needed here? `Name` is declared non-nullable, and `ToProperCase()` does not return null if the parameter is not null.

Reply:
Nope, not needed. This was before I added the annotations.